src/TextTagger.php

```diff
@@ -0,0 +1,135 @@
+<?php
+namespace Whatafix\TextTagger;
+use Whatafix\TextTagger\Contracts\TextTaggerInterface;
+class TextTagger implements TextTaggerInterface
+{

```

Comment:
Attention au CS (code style) ça ne change pas le fonctionnement mais c'est bien de respecter les usages.
```suggestion
<?php

namespace Whatafix\TextTagger;

use Whatafix\TextTagger\Contracts\TextTaggerInterface;

class TextTagger implements TextTaggerInterface
{

```